packages/create-block/CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 ## Master
 
+### Enhancements
+
+- Removed the code that clears the terminal while the block is scaffolded ([#19867](https://github.com/WordPress/gutenberg/pull/19867)).
+
+### Bug Fixes
+
+- Use the description provided to fill the `description` field in `package.json` file in ESNext template ([#19867](https://github.com/WordPress/gutenberg/pull/19867)).
+- Ensure that values provided for slug and namespace get converted to lower case ([#19867](https://github.com/WordPress/gutenberg/pull/19867)).
+
```

Comment:
I guess it can be good to be exhaustive, but technically since we've never published a version before, there's nothing that anyone needs to concern over in the CHANGELOG ?

Reply:
I'm guessing @gziolo will deprecate his package and add a notice linking to this package

Reply:
Indeed, I want to deprecate the old package as soon as this one is published to npm. I wanted to make it clear what has changed between the import and publish. There are two options. I remove those additional  lines or I copy old changelog for full history. What do you think?

Reply:
I'd copy across the [changelog](https://github.com/gziolo/create-wordpress-block/blob/master/CHANGELOG.md) and also add an addition entry highlighting that the package relocated repos for the changelog for this release

Reply:
I followed your suggestion: b6d87122f34e0dccfe19f23fbb6593c8b58a2ce5.